var/spack/repos/builtin/packages/dbcsr/package.py

```diff
@@ -65,7 +67,7 @@ class Dbcsr(CMakePackage, CudaPackage, ROCmPackage):
     # for optimal kernels. Note that we don't override the parent class arch
     # properties, since the parent class defines constraints for different archs
     # Instead just mark all unsupported cuda archs as conflicting.
-    dbcsr_cuda_archs = ("35", "37", "60", "70")
+    dbcsr_cuda_archs = ("35", "37", "60", "70", "80")
```

Comment:
I guess `80` is valid only from 2.6.0 onwards?

Reply:
indeed